projects/driver_controls/inc/ads1015_def.h

```diff
@@ -56,6 +56,8 @@
 #define ADS1015_DATA_RATE_2400 ((uint8_t)0x5 << 5)  // 2400 SPS
 #define ADS1015_DATA_RATE_3300 ((uint8_t)0x6 << 5)  // 3300 SPS
 
+#define ADS1015_CONVERSION_TIME_US_1600_SPS (0xF4240 / 1600)
```

Comment:
Where did this number come from?

Reply:
That is a second in microseconds so a million. I will add another macro for it.

Reply:
There's no need to do that. The hex number was just weird. You can just keep this inside of the source file.